index.js

```diff
@@ -129,6 +149,7 @@ BlindsHTTPAccessory.prototype.setTargetPosition = function(pos, callback) {
             self.stepInterval = setInterval(function() {
                 if (self.lastPosition == self.currentTargetPosition) {
                     self.log(`End ${moveMessage} (to ${self.currentTargetPosition}%)`);
+                    // TODO: modify behavior based on self.positionURL
```

Comment:
do you plan on implementing this in this pull request?

Reply:
I left this as a note, because probably we need to implement:

`BlindsHTTPAccessory.prototype.setCurrentPosition`

And intercept it.

In this case, I was thinking maybe the request to fetch the position from the URL should probably be a separate function.  I'm also not sure how polling by HomeKit works, if we need to poll for the currentPosition, and also probably this should be implemented and we can implement it here too:

https://github.com/zwerch/homebridge-blinds/blob/29025d012cec2fc8c64bbdfd408911148a15a59c/index.js#L44

I didn't get to it, because it's more of a new feature request and there are probably better ways to implement it, but feel free to take a crack at it in the meantime :)

I've found some helpful and relevant examples of ways to improve things here:
- https://github.com/Nicnl/homebridge-minimal-http-blinds
- https://github.com/Supereg/homebridge-http-switch

I like some of the regex matching and also more generic ways of expressing the URL (as a single config parameter that can be optionally turned into an object).